contracts/pm/mixins/MixinTicketBrokerCore.sol

```diff
@@ -114,6 +117,8 @@ contract MixinTicketBrokerCore is MContractRegistry, MReserve, MTicketProcessor,
         uint256 _recipientRand
     )
         public
+        currentRoundInitialized
```

Comment:
Let's run the `whenSystemNotPaused()` check first - this is the ordering we use in the BondingManager and also skips the round initialized check when the system is actually paused.

Reply:
fixed in 752bec9